Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/DataController.swift

```diff
@@ -267,6 +267,14 @@ final class DataController: NSObject, WCSessionDelegate {
         }
     }
     
+    func sessionReachabilityDidChange(session: WCSession) {
+        print("iOS application reachability changed.")
```

Comment:
Yes it is debug only, just in the example project, not present in the wrapper framework.